src/lists/_base.scss

```diff
@@ -5,5 +5,5 @@
  */
 
 .list {
-  padding: ms(1) 0;
+  margin: ms(1) 0;
```

Comment:
@josephco also brought this up, but we may just want to standardize on bottom or top that way we don't need to do any overrides for final elements. Maybe this isn't an issue for this module. Fit and finish could shed some more insight.